environment

```diff
@@ -13,7 +13,7 @@ if [[ $STAGE == "prod" ]]; then
 else
     API_DOMAIN_NAME=query.${STAGE}.data.humancellatlas.org
     API_DNS_ZONE=${STAGE}.data.humancellatlas.org.
-    DSS_HOST=dss.${STAGE}.data.humancellatlas.org
+    DSS_HOST=dss.staging.data.humancellatlas.org
```

Comment:
should we add a comment about why this is being hardcoded to staging?

Reply:
Good idea, added a comment.